debian/changelog

```diff
@@ -1,3 +1,116 @@
+cloud-init (20.2-0ubuntu1) gorilla; urgency=medium
+
+  * drop the following cherry-picks now included:
+    + cpick-6600c642-ec2-render-network-on-all-NICs-and-add-secondary-IPs-as
+    + cpick-986f37b0-cloudinit-move-to-pytest-for-running-tests-211
+    + cpick-4fb6fd8a-net-ubuntu-focal-prioritize-netplan-over-eni-even-if
+    + cpick-04771d75-cc_disk_setup-fix-RuntimeError-270
+    + cpick-c5e949c0-distros-tests-test_init-add-tests-for
+    + cpick-2566fdbe-net-introduce-is_ip_address-function-288
+    + cpick-4f825b3e-cloudinit-refactor-util.is_ipv4-to-net.is_ipv4_address
+    + cpick-c478d0bf-distros-replace-invalid-characters-in-mirror-URLs-with
+    + cpick-1bbc4908-distros-drop-leading-trailing-hyphens-from-mirror-URL
+    + cpick-09fea85f-net-ignore-renderer-key-in-netplan-config-306
+    + cpick-9d7b35ce-cc_mounts-fix-incorrect-format-specifiers-316
+    + cpick-0c5c7367-test_mounts-expand-happy-path-test-for-both-happy-paths
+  * refresh patches:
+   + debian/patches/fix-cpick-4fb6fd8a-net-ubuntu-focal-prioritize-netplan-over-eni.patch
```

Comment:
I think this patch should be dropped instead of refreshed, as its header says:

> Support for netbsd and openbsd is on in upstream until new-upstream-snapshot.

And we are now new-upstream-snapshot'ing.

Reply:
Good point, this should be dropped, and that drop bevior is included in https://github.com/canonical/uss-tableflip/pull/45
Maybe we work to land https://github.com/canonical/uss-tableflip/pull/45 first so we don't have to manually fix this. and I can then open a new PR that is a full new-upstream-snapshot.